### ISSUE TITLE
ModuleNotFoundError: No module named 'tflite_runtime' [Windows]

### ISSUE THREAD

**Issue by FuryBart** [kahst/BirdNET-Analyzer#346]
**Describe the bug**
When I start the BirdNET-Analyzer.exe, I get an error message.
_ModuleNotFoundError: No module named 'tflite_runtime'_

**To Reproduce**
I have downloaded the latest version: BirdNET-Analyzer-GUI-1.x.x-win.exe, unblocked the exe file and followed the installation instructions https://github.com/kahst/BirdNET-Analyzer?tab=readme-ov-file#setup-windows.

As described in the instructions, I did not install Python or Tensorflow separately as it is a full version with no additional packages to install.

**Expected behavior**
Starting BirdNET Analyzer.

**Desktop**
 - OS: Windows 10 22H2 (latest CU)
 - Versions 1.0.2, 1.1.0, 1.2.0, 1.3.0

**Complete Error Message**
```
Traceback (most recent call last):
  File "model.py", line 21, in <module>
ModuleNotFoundError: No module named 'tflite_runtime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "analyze.py", line 17, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "model.py", line 23, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "tensorflow\__init__.py", line 45, in <module>
    from tensorflow.python import tf2 as _tf2
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "tensorflow\python\tf2.py", line 21, in <module>
    from tensorflow.python.platform import _pywrap_tf2
ImportError: DLL load failed while importing _pywrap_tf2: Eine DLL-Initialisierungsroutine ist fehlgeschlagen.
[9500] Failed to execute script 'analyze' due to unhandled exception!
```

**Comment by kevtolan** [comment]
This is the same issue [I'm running into](https://github.com/kahst/BirdNET-Analyzer/issues/345) in Python. 

Runtime was missing from my package list. I tried installing it via `pip3 install tflite_runtime` and got this error:
```ktol@KTolVCE BirdNET-Analyzer % pip3 install tflite_runtime
ERROR: Could not find a version that satisfies the requirement tflite_runtime (from versions: none)
ERROR: No matching distribution found for tflite_runtime
```

next I tried via `pip3 install --extra-index-url https://google-coral.github.io/py-repo/ tflite_runtime`, which was successful in downloading, and it appeared in my conda list: `tflite_runtime-2.5.0.post1`


I tried running `python analyze.py` again, and got the same error, despite runtime being in the environment.
```
    import tflite_runtime.interpreter as tflite
ModuleNotFoundError: No module named 'tflite_runtime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/kevintolan/BirdNET-Analyzer/analyze.py", line 17, in <module>
    import model
  File "/Users/kevintolan/BirdNET-Analyzer/model.py", line 23, in <module>
    from tensorflow import lite as tflite
ImportError: cannot import name 'lite' from 'tensorflow' (unknown location)
(birdnet) ktol@KTolVCE BirdNET-Analyzer % 
```

**Comment by FuryBart** [comment]
Update. I have tested the first release BirdNET-Analyzer-GUI-1.0.2-win.exe with Windows 10 [22H2 19045.4474] and got the same error. Installing the latest VC distributable dll's from 
https://learn.microsoft.com/de-de/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version had no effect. Something must be missing in the "as-is" package.

Can I provide any additional information?

**Comment by FuryBart** [comment]
Unfortunately I've the same issue with the latest release 1.2.0. _ModuleNotFoundError: No module named 'tflite_runtime'._

In the PDF [BirdNET-Analyzer Usage Guide](https://zenodo.org/records/8357176) I found a download Link to an older version of a BirdNET-Analyzer.zip from tuc.cloud. This version doesn't need any installation an runs as it is. Maybe this information is somehow helpful.

**Comment by Josef-Haupt** [comment]
This seems to be the same problem listed in #543 
TLDR: We require AVX2 insctructions but seems like your CPU doen't have that.
